### DISCUSSION TITLE
Reusing redis container as system redis server

### DISCUSSION THREAD

**Discussion by Sid127** [nextcloud/all-in-one#2409] [Questions] [answered]
I'm hosting Nextcloud AIO and also want to start hosting sourcehut, I was wondering if it'd be possible to reuse the existing redis container that NC-AIO has with sourcehut local install

**Comment by szaimen** [comment, marked as answer]
Hi, redis is very ressource efficient, so I would recommend to simply create a new redis instance for sourcehut which properly isolates the different projects from each other.

**Reply by Sid127** [reply]
okay, thank you very much :)
